.github/workflows/ci.yml

```diff
@@ -581,6 +569,140 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
 
+  azure-integration-tests:
+    name: AzureDB IT
+    needs: conditional
+    if: needs.conditional.outputs.ci == 'true'
+    runs-on: ubuntu-latest
+    timeout-minutes: 150
+    permissions:
+      contents: read
+      actions: write
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - id: node-cache
+        name: Node cache
+        uses: ./.github/actions/node-cache
+
+      - id: azure-init
+        name: Initialize Azure environment
+        run: |
+          AZURE_REGION=westeurope
+          echo "AZURE Region: ${AZURE_REGION}"
+
+          AZURE_SUBSCRIPTION=${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+          AZURE_CLUSTER_NAME="gh-action-$(git rev-parse --short HEAD)-${{ github.run_id }}-${{ github.run_attempt }}"
+          echo "azure-cluster-name=${AZURE_CLUSTER_NAME}" >> $GITHUB_OUTPUT
+          echo "region=${AZURE_REGION}" >> $GITHUB_OUTPUT
+          echo "azure-subscription=${AZURE_SUBSCRIPTION}" >> $GITHUB_OUTPUT
```

Comment:
Should we use the secret directly in steps instead of passing it as a step output?